ocrd/ocrd/processor/base.py

```diff
@@ -154,25 +154,11 @@ def resolve_resource(self, val):
             val (string): resource value to resolve
         """
         executable = self.ocrd_tool['executable']
-        if exists(val):
-            return val
```

Comment:
Why'd you remove resolving absolute path names?

Reply:
:blush: accident. fixing.

Reply:
d5443c2 fixed